server/sonar-server/src/main/java/org/sonar/server/user/AbstractUserSession.java

```diff
@@ -141,6 +143,16 @@ public UserSession checkGlobalPermission(String globalPermission, @Nullable Stri
   }
 
   @Override
+  public UserSession checkAnyGlobalPermissions(Collection<String> globalPermissions) {
+    Collection intersection = CollectionUtils.intersection(globalPermissions, globalPermissions());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) org.sonar.server.user.AbstractUserSession must not use org.apache.commons.collections.CollectionUtils [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3Acommons_collections_should_not_be_used)